Chapter 1 and 2 corrections (#99)
chhoumann/p8: report/src/problem_analysis/understanding.tex

The goals of these interviews were twofold: 
First and foremost, we wanted to understand the current issues the stakeholders are facing. 
To accommodate this, an unstructured interview was used to explore the domain and problem~\cite{benyon2013designing}. 
This form of interview is beneficial when exploring the domain, as unexpected findings and new perspectives are often found when exploring the situation with the users\cite{benyon2013designing}.
Secondly, we wanted to understand what the stakeholder sees as minimum requirements for a potential solution to their problem. 
To reach this understanding, a semi-structured interview was conducted after an initial, exploratory interview.

> Due to a lacking understanding of all of the specifics of the stakeholders' problems, creating close-ended questions for a structured interview would not be ideal.

Reviewer: ```suggestion
Given our incomplete understanding of the full range of stakeholder issues, formulating closed-ended questions for a structured interview would not have been be optimal.
```

We therefore chose to use a semi-structured interview focusing on open-ended questions. This allowed us to ask complementary questions to clarify answers \cite{InterviewsNHS}.
The basis for this interview was the initial ideas and concepts related to the functionality and UI provided to us by the stakeholder during the preliminary engagement.

> During this interview, the stakeholders provided us with insight into what their current needs are, and why their current system does not satisfy these needs.

Reviewer: ```suggestion
During this interview, the stakeholders provided insights into what their needs were, and why their current system do not satisfy these.
```

> First and foremost, the stakeholder has issues with arranging meetings throughout their day.

Reviewer: ```suggestion
First, the stakeholder has issues with arranging meetings throughout their day.
```

The user stories where then sent to the stakeholders and their validity confirmed.  
The validated user stories can be seen in Appendix \ref{appendix:user_stories}.

> After having conducted a preliminary analysis of the stakeholders' impediments, we embarked on the process of trying to find a common denominator for these problems.

Reviewer: ```suggestion
After having conducted a preliminary analysis of the stakeholders' impediments, we tried to find a common denominator for these problems.
```
